index.js

```diff
@@ -86,72 +84,57 @@ Collector.prototype._notifyIfFlushed = function(callback, err) {
     setImmediate(callback, err);
 };
 
-Collector.prototype._flushSeries = function(seriesName, points, callback) {
+Collector.prototype._flushPoints = function(points, callback) {
     if (!points || points.length === 0) {
         return;
     }
     var self = this;
 
     // only send N points at a time to avoid making requests too large
-    var spliceIndex;
-    if (self._client.protocol == 'udp:' || self._client.protocol == 'line') {
-      spliceIndex = self.computePointCountToSend(points.map(function (point) {
-        return JSON.stringify(point).length;
-      }), MTU_SIZE);
-    } else {
-      spliceIndex = 50;
-    }
+    var spliceIndex = 50
```

Comment:
We should look into increasing this value since the `node-influx` lib docs suggest bundling thousands of points at a time (https://node-influx.github.io/class/src/index.js~InfluxDB.html#instance-method-writePoints). Is there any particular reason it was previously set to such a low value?

`computePointCountToSend` was previously used to calculate the batch size when using UDP, but I've left it in here in case we want to use it to dynamically compute the batch size when using TCP also.